apps/designer/app/designer/features/breakpoints/breakpoints.tsx

```diff
@@ -44,17 +43,17 @@ const BreakpointSelectorItem = ({
 }: BreakpointSelectorItemProps) => {
   return (
     <Flex align="center" justify="between" gap="3" css={{ flexGrow: 1 }}>
-      <__DEPRECATED__Text size="1" css={{ flexGrow: 1, color: "inherit" }}>
+      <Text size="1" css={{ flexGrow: 1, color: "inherit" }}>
```

Comment:
how did it pass tsc? size prop doesn't exist on new Text component

Reply:
@okonet should Text have flexGrow: 1 and color inherit by default baked in?

Reply:
@okonet should we add a variant for color="inherit"? <Text color="inherit" /> in addition to those others

Reply:
Actually regarding color there might be something wrong, what is the color that is used there? Do existing color variants not work?

Reply:
It's hard to get the default for Text right. Sometimes I tend to use `color: inherit` as a default value so it can be composed easily, so it could make sense. On the other hand, what's not working with the color defined in the variant? Why is it needs to be overridden in this case?

As for flex, what I usually prefer is to make components take all available space and handle the layout outside of them. Would it work here?

I also think that Text should not be `block` or `flex` but instead `inline-block` or `inline-flex` since you have `Text` nested with `Text`. Right now it's not possible to do afaik.